doc/install.text

```diff
@@ -0,0 +1,124 @@
+eups declare drp_stella_data 1.0 -c -f Linux64 -r ~/stella-git/drp_stella_data
+eups declare drp_stella 1.0 -c -f Linux64 -r ~/stella-git/drp_stella
+
+
+* Setup the pipeline. As obs_pfs and drp_stella_data are required packages they are set up automatically:
+
+setup drp_stella 1.0 -v
+
+During this step the environment variables $DRP_STELLA_DIR, $DRP_STELLA_DATA_DIR, and $OBS_PFS_DIR are automatically 
+created as well:
+
+echo $DRP_STELLA_DIR
+
+should show you:
+
+~/stella-git/drp_stella
+
+
+* Build the pipeline:
+
+cd drp_stella
+scons opt=3 -j8
+
+
+* Create the defects database (done during compilation of the code):
+
+cd ../obs_pfs
+scons opt=3 -j8
+
+
+* Now for using the pipeline. Raw test data are in $DRP_STELLA_DATA_DIR/tests/data/raw/ (3 Biases, 3 Darks, 1 Flat, 1 Arc). 
+First we need to create a directory (actually 2) where we want to store pipeline outputs. 
+Let's assume you have a directory ~/spectra and do everything related to spectra there:
+
+mkdir ~/spectra/PFS
+mkdir ~/spectra/PFS/CALIB
+
+
+* Configuration parameters can be passed to the pipeline task either on the command line or in a config file 
+(e.g. $OBS_PFS_DIR/config/pfs/ingest.py, see following example). You can list all possible configuration parameters by 
+appending a "--show config" to the parameter list.
+To ingest the raw images:
+
+cd ~/spectra
+ingestImages.py PFS $DRP_STELLA_DATA_DIR/tests/data/raw/*.fits --output=PFS --mode=link -C $OBS_PFS_DIR/config/pfs/ingest.py
+
+
+* You can now inspect the created SQL database with e.g. sqlitebrowser (http://sqlitebrowser.org/).
+Now that we have our database we can start reducing things. We should probably start with the Biases. If the biases we
+want to reduce were observed on 2015-12-22 on spectrograph 2, arm red (r) at site S:
+
+reduceBias.py PFS --calib PFS/CALIB --output PFS --calibId calibVersion=bias arm=r calibDate=2015-12-22 spectrograph=2 --do-exec --id field=BIAS dateObs=2015-12-22 --clobber-config --nodes=1 --procs=1
+
+
+* Note the 2 config parameters --nodes and --procs at the end. These parameters are required by tasks which are 
+parallelized. Sometimes running the code in parallel can lead to problems (in most cases caused by the 3rd-party libraries
+used), so setting nodes and procs to 1 is a safe choice. —-clobber-config is needed if you re-run the task with different
+config parameters. The config used for each run is written to PFS/config/bias.py and the task won’t run again if you 
+change parameters, unless you say —-clobber-config.
+
+* Now that we have a master bias we need to inject that too into our database:
+
+genCalibRegistry.py --root=PFS/CALIB --camera PFS --validity 180
+
+* Now we can create a bias-subtracted Dark and ingest that into our database:
+
+reduceDark.py PFS --calib PFS/CALIB --output PFS --calibId calibVersion=dark arm=r calibDate=2015-12-22 spectrograph=2 --do-exec --id field=DARK dateObs=2015-12-22 --nodes=1 --procs=1 -C ~/stella-git/obs_pfs/config/pfs/dark.py
```

Comment:
There is a gotcha here.

We have been investigating simulated spots for the NIR arm; those are the most challenging, and the ones we worry most about understanding.

But the simulated & real images that we currently have are for/from CCDs. Specifically one of the red ones. Since a) we had real red CCD images and b) we understand the ISR components for HSC CCDs it made sense to simulate the NIR spots onto those CCD "detectors".

Yes we will have an ISR for the IR ramps, but we don't have one yet. So on the command line <arm=n> is probably OK, but the raw images really are quite different from what the <arm=n> ramp files will look like.

I think that in the end we will want to remove all traces of this confusing little fib. In the meanwhile it is probably better to use <arm=r> in as many places as possible.